main.js

```diff
@@ -1 +1,36 @@
 'use strict';
+let sorting = document.getElementById('sorting');
+
+sorting.onclick = function(e) {
+  if (e.target.tagName != 'TH') return;
+  sortGrid(e.target.cellIndex, e.target.getAttribute('data-type'));
+};
+
+function sortGrid(colNum, type) {
+  let tbody = sorting.getElementsByTagName('tbody')[0];
+  let rowsArray = [].slice.call(tbody.rows);
```

Comment:
will be better to call `slice.call` on `Array.prototype` instead of creating an array literal